src/main/scripts/travis-build.bash

```diff
@@ -76,6 +76,11 @@ function main() {
             err "failed to push git tags"
             return 1
         fi
+
+        if ! $mvn package docker:build -DskipTests -DpushImageTag; then
```

Comment:
It will get built and pushed on every master build and "release". If you only want it on release, put a conditional around the docker build.

```sh
if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
    if ! $mvn package docker:build ...
```